src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AzureServiceBus/AzureServiceBusReceiverManagerWrapper.cs

```diff
@@ -25,11 +29,26 @@ public override CanWrapResponse CanWrap(InstrumentedMethodInfo instrumentedMetho
         public override AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall instrumentedMethodCall, IAgent agent, ITransaction transaction)
         {
             var receiverManager = instrumentedMethodCall.MethodCall.InvocationTarget;
-            _receiverAccessor ??= VisibilityBypasser.Instance.GeneratePropertyAccessor<object>(receiverManager.GetType(), "Receiver");
-            dynamic receiver = _receiverAccessor(receiverManager);
 
-            string queueOrTopicName = receiver.EntityPath; // some-queue|topic-name
-            string fqns = receiver.FullyQualifiedNamespace; // some-service-bus-entity.servicebus.windows.net
+            var receiverManagerType = receiverManager.GetType();
+            _receiverAccessor = VisibilityBypasser.Instance.GeneratePropertyAccessor<object>(receiverManagerType, "Receiver");
+            object receiver = _receiverAccessor(receiverManager);
+            if (receiver == null)
+            {
+                if (!_badReceiverWarningLogged)
+                {
+                    agent.Logger.Warn("AzureServiceBusReceiverManagerWrapper: Unable to access Receiver property on ReceiverManager instance of type {ReceiverManagerType}. Unable to instrument Azure ServiceBus Processor Mode.", receiverManagerType);
+                    _badReceiverWarningLogged = true;
+                }
+                return Delegates.NoOp;
```

Comment:
Maybe we don't need to NoOp here, and can just let queueOTopicName and fqns be null.